docs/source/guide/release_notes.md

```diff
@@ -3,9 +3,8 @@ title: Release notes for Label Studio Enterprise
 short: Release notes
 badge: <i class='ent'/></i>
 type: guide
-tier: all
-order: 120
-section: "Install"
+tier: 
+order: 
```

Comment:
Maybe it works with a number here, but you should add one just in case

Reply:
<img width="1478" alt="Screen Shot 2022-11-17 at 8 45 25 PM" src="https://user-images.githubusercontent.com/107003313/202618884-97792b17-66f3-41a1-bf1e-86e37b11b8fc.png">